lib/plugins/milestones.js

```diff
@@ -0,0 +1,37 @@
+const Diffable = require('./diffable')
+
+module.exports = class Milestones extends Diffable {
+  find () {
+    let allMilestones = []
+
+    allMilestones.push(this.github.issues.getMilestones(Object.assign({}, {state: 'open'}, this.repo)).then(res => res.data))
+    allMilestones.push(this.github.issues.getMilestones(Object.assign({}, {state: 'closed'}, this.repo)).then(res => res.data))
```

Comment:
This can be consolidated to one request with `{state: 'all'}`. See https://developer.github.com/v3/issues/milestones/#list-milestones-for-a-repository

Reply:
Also, instead of `then`, most code in Probot uses `async`/`await`:

```js
allMilestones.push((await this.github.issues.getMilestones(params)).data)
```